src/Microsoft.ML.Data/Model/Onnx/SaveOnnxCommand.cs

```diff
@@ -68,18 +83,28 @@ public SaveOnnxCommand(IHostEnvironment env, Arguments args)
                 _name = args.Name;
 
             _loadPredictor = args.LoadPredictor;
-            _inputsToDrop = CreateDropMap(args.InputsToDrop);
-            _outputsToDrop = CreateDropMap(args.OutputsToDrop);
+            _inputsToDrop = args.InputsToDropArray != null ? CreateDropMap(args.InputsToDropArray) : CreateDropMap(args.InputsToDrop);
+            _outputsToDrop = args.OutputsToDropArray != null ? CreateDropMap(args.OutputsToDropArray) : CreateDropMap(args.OutputsToDrop);
             _domain = args.Domain;
+            _model = args.Model;
         }
 
         private static HashSet<string> CreateDropMap(string toDrop)
         {
             if (string.IsNullOrWhiteSpace(toDrop))
                 return new HashSet<string>();
+
             return new HashSet<string>(toDrop.Split(','));
```

Comment:
With the introduction, we can replace with `return CreateDropMap(toDrop.?Split(','));`.